docs/decisions/001-strip-whitespaces.md

```diff
@@ -11,7 +11,14 @@
 
 ## Decision
 
-In order to avoid issues related to whitespaces, we should strip whitespaces between HTML tags by default and only add them when needed. The Twig template is considered valid when the output it produces doesn't contain any whitespace between 2 HTML tags or between an HTML tag and its text content, unless the whitespace is explicitly required.
+In order to avoid issues related to whitespaces, we should strip whitespaces between HTML tags by default and only add them when needed.
+
+The Twig template is considered valid when the output it produces doesn't contain any whitespace:
+
+- between 2 HTML tags ;
```

Comment:
I personally feel these 2 rules/guidelines are already a good base for being able to move forward with the document and it's implementation.